app/src/main/java/com/zulip/android/ZulipApp.java

```diff
@@ -261,4 +285,21 @@ public void markMessageAsRead(Message message) {
             unreadMessageHandler.sendEmptyMessageDelayed(0, 2000);
         }
     }
+    public void muteTopic(Message message) {
+        mutedTopics.add(message.getStream().getId() + message.getSubject());
```

Comment:
@niftynei 
I think the stream won't be null as these are only for the muting topics are only for the stream messages!